docs/sources/dashboards/share-dashboards-panels/shared-dashboards/index.md

```diff
@@ -0,0 +1,327 @@
+---
+aliases:
+  - ../dashboard-public/ # /docs/grafana/latest/dashboards/dashboard-public/
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+title: Externally shared dashboards
+menuTitle: Shared dashboards
+description: Make your Grafana dashboards externally shared and share them with anyone
+weight: 8
+refs:
+  dashboard-sharing:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/dashboards/share-dashboards-panels/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/visualizations/dashboards/share-dashboards-panels/
+  custom-branding:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/configure-custom-branding/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/configure-custom-branding/
+  dashboard-insights-documentation:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/dashboards/assess-dashboard-usage/#dashboard-insights
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/visualizations/dashboards/assess-dashboard-usage/
+  caching:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/administration/data-source-management/#query-and-resource-caching
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana/<GRAFANA_VERSION>/administration/data-source-management/#query-and-resource-caching
+  grafana-enterprise:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/introduction/grafana-enterprise/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana/<GRAFANA_VERSION>/introduction/grafana-enterprise/
+---
+
+# Externally shared dashboards
+
+{{< admonition type="warning" >}}
+Sharing your dashboard externally could result in a large number of queries to the data sources used by your dashboard.
+This can be mitigated by using the Enterprise [caching](ref:caching) and/or rate limiting features.
+{{< /admonition >}}
+
+Externally shared dashboards allow you to share your Grafana dashboard with anyone. This is useful when you want to make your dashboard available to the world without requiring access to your Grafana organization.
+
+{{< admonition type="note" >}}
+If you change a dashboard, ensure that you save the changes before sharing.
+{{< /admonition >}}
+
+## Shared dashboards list
+
+You can see a list of all your externally shared dashboards in one place by navigating to **Dashboards > Shared dashboards**. For each dashboard in the list, the page displays:
+
+- Link to view the externally shared version of the dashboard
+- Link to the shared dashboard configuration
+- Options to pause or revoke access to the external dashboard
+
+You can also click the name of the dashboard to navigate to the dashboard internally.
+
+## Security implications of sharing your dashboard externally
```

Comment:
I know the word "implication" doesn't mean just risks, but I definitely expected to see more risks in this section vs things that seem actually pretty positive from a security perspective. Suggest something less scary.
```suggestion
## Important notes about sharing your dashboard externally
```